### DISCUSSION TITLE
plugins dont working on start

### DISCUSSION THREAD

**Discussion by zibertszn** [ohmyzsh/ohmyzsh#10627] [Help] [answered]
plugins dont working on terminal start , but when i write "source ~/.zshrc" . plugins start working

**Comment by mcornella** [comment, marked as answer]
It's probable that the line defining plugins is written below the line sourcing OMZ. Check your zshrc file for the correct order:

## ❌ This does not work

```zsh
...
source $ZSH/oh-my-zsh.sh
...
plugins=(enabled plugins go here)
```

## ✅ It should look like this

```zsh
...
plugins=(enabled plugins go here)
...
source $ZSH/oh-my-zsh.sh
```

**Reply by DavidInRacine** [reply]
The answer was helpful to this new OMZ user.  Thanks so much!

**Comment by Tamarwish** [comment]
Hello
